evaluation/README.md

```diff
@@ -46,28 +46,32 @@ The OpenHands evaluation harness supports a wide variety of benchmarks across so
 
 ### Software Engineering
 
-- SWE-Bench: [`evaluation/swe_bench`](./swe_bench)
-- HumanEvalFix: [`evaluation/humanevalfix`](./humanevalfix)
-- BIRD: [`evaluation/bird`](./bird)
-- BioCoder: [`evaluation/ml_bench`](./ml_bench)
-- ML-Bench: [`evaluation/ml_bench`](./ml_bench)
-- APIBench: [`evaluation/gorilla`](./gorilla/)
-- ToolQA: [`evaluation/toolqa`](./toolqa/)
-- AiderBench: [`evaluation/aider_bench`](./aider_bench/)
+- SWE-Bench: [`evaluation/benchmarks/swe_bench`](./benchmarks/swe_bench)
```

Comment:
If I see this right, the agent moved the files, but did not update any code, for example this?
https://github.com/All-Hands-AI/OpenHands/blob/4c621966d489f44d3a1ddd167901a6885caba914/evaluation/benchmarks/swe_bench/eval_infer.py#L15

Reply:
Thanks, should be fixed!